app/graphql/schema.graphql

```diff
@@ -545,6 +546,17 @@ type SystemEdge {
   node: System
 }
 
+"""
+A host inventory tag
+"""
+type Tag implements Node {
+  globalId: ID!
```

Comment:
Where/how is the `globalId` provided?

Reply:
It's been auto generated :shrug: do you know how can I get rid of it? I guess some magic declaration in the type def :thinking:

Reply:
Okay, I think I have it...will figure this out